runtime/src/invoke_dpe.rs

```diff
@@ -91,7 +109,32 @@ impl InvokeDpeCmd {
         }
     }
 
-    fn is_caller_p1(pl0_pauser: u32, flags: u32, locality: u32) -> bool {
+    fn pl_context_threshold_exceeded(
```

Comment:
Code looks good. Can you also add a section to the runtime spec (somewhere near the other DPE details) explaining that we're doing this and why?

Reply:
Done.